.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```

Comment:
the .idea folder should not be included

Reply:
it was mentioned in a previous comment to add .idea folder to .gitignore.Can you please reconfirm if you are asking to remove the .idea that I added  from .gitignore file.

Reply:
It is ok that the .idea item has been added to the .gitignore
It is only necessary now to untrack the corresponding files from it because they were added to the initial commit